pypeit/specobj.py

```diff
@@ -80,16 +80,23 @@
     'VEL_TYPE': dict(otype=str, desc='Type of heliocentric correction (if any)'),
     'VEL_CORR': dict(otype=float, desc='Relativistic velocity correction for wavelengths'),
     #
-    'DET': dict(otype=(int,np.int64), desc='Detector number'),
+    'DET': dict(otype=(int,np.int64,np.int32), desc='Detector number'),
```

Comment:
Use `(int, np.integer)` instead?

E.g.:
```
>>> x = numpy.int64(3)
>>> y = numpy.int32(3)
>>> z = 3
>>> type(z)
<class 'int'>
>>> isinstance(x, numpy.integer)
True
>>> isinstance(y, numpy.integer)
True
>>> isinstance(z, numpy.integer)
False
```

Reply:
Is there an analog for this for float 32 and float 64. np.float does not work??

Reply:
Yeah, `np.floating`.

```
>>> import numpy
>>> x = numpy.float64(2.2)
>>> y = numpy.float32(2.2)
>>> z = 2.2
>>> type(z)
<class 'float'>
>>> isinstance(x, numpy.floating)
True
>>> isinstance(y, numpy.floating)
True
>>> isinstance(z, numpy.floating)
False
```